### PR TITLE
Change projection days from a slider to a number input to allow larger max values

### LINKED ISSUE #199: Expand max days past 200
> We are a regional health system in northeast ohio and we have been using your modeling to help predict our surge.  Question, when we do six days as the doubling number the graph at 200 days is still going up.  Can you help to expand the days so we can see the full shape of the curve?

Let's set the max to 400 days and see if that addresses these issues.

### PR DESCRIPTION
**Pull request by nickcanz**
This PR changes the "Number of days to project" input from a slider to a number input. This allows users to set an exact value and to values greater than 200.

[Some context about slider vs input from Slack](https://codeforphilly.slack.com/archives/CV71EUYD8/p1584985223323200)

Closes #199

From comments, I also added to this PR to move the last few presentation UI elements to display in the sidebar and into the `Parameters` model.

Full screenshot of the changed UI.
Screen Shot 2020-03-23 at 15 03 29

### CODE REVIEW

**Comment by nickcanz** [comment]
Screenshot of what it looks like as an input instead of a slider.
Screen Shot 2020-03-23 at 14 24 25

**Comment by cjbayesian** [comment]
Probably should go in the sidebar for consistency with all the other inputs.

**Comment by nickcanz** [comment]
@cjbayesian I'm cool moving to the sider bar.

1. Should it be the first value?
2. Move the "display as date" checkbox to the siderbar as well?

File: src/app.py [hunk 1]
```diff
@@ -7,7 +7,6 @@
     # additional_projections_chart,
```
Comment (quinn-dougherty) [inline src/app.py]: do me a favor and remove this commented out import please?

Comment (nickcanz) [reply src/app.py]: 👍 done

**Review by quinn-dougherty — APPROVED** [review]

**Pull request merged**